pages/agent/v3/docker.md.erb

```diff
@@ -134,7 +134,7 @@ docker run \
   buildkite/agent:3 start --token "<your-agent-token>"
 ```
 
-You can then use an `environment` [agent hook](/docs/agent/hooks) to expose those secrets via environment variables, or to configure tools such as `git` or `ssh`.
+You can then use an `environment` [agent hook](/docs/agent/hooks) to expose those secrets via environment variables, or to configure tools such as `git` or `ssh`. Please ensure that you do not to print or export these variables anywhere in your pipeline. For more information on secret management, see the [Secrets](/docs/pipelines/secrets) documentation.
```

Comment:
I think we might need to remove this paragraph altogether, and replace it with something like:

> If you've exposed [pipeline secrets](/docs/pipelines/secrets) as environment variables, you can pass them through to the container using the `-e` option:
> 
> ```bash
> docker run \
>   -e MY_SECRET_ENV \
>   -d \
>   -t \
>   --name buildkite-agent \
>   buildkite/agent:3 start --token "<your-agent-token>"
> ```